File: my_commands/in_command.py
#in_command
from my_commands.in_commands.aaaaa import aaaaa 
from my_commands.in_commands.good_night import good_night 
from my_commands.in_commands.heysiri import heysiri 
from my_commands.in_commands.LEGEND import LEGEND 
from my_commands.in_commands.mohumohu import mohumohu 
from my_commands.in_commands.poponta import poponta 
async def in_command(message):
    if 'ﾌﾞﾘﾌﾞﾘﾌﾞﾘﾌﾞﾘｭﾘｭﾘｭﾘｭﾘｭﾘｭﾌﾞﾂﾁﾁﾌﾞﾌﾞﾌﾞﾁﾁﾁﾁﾌﾞﾘﾘｲﾘﾌﾞﾌﾞﾌﾞﾌﾞｩｩｩｩｯｯｯ' in message.content:
        await aaaaa(message)
    if 'おやすみ' in message.content:
        await good_night(message)
    if 'Hey Siri' in message.content:
        await heysiri(message)
    if 'LEGEND' in message.content:
        await LEGEND(message)
    if 'もふもふ' in message.content:
        await mohumohu(message)
    if chr(129300) in message.content:
        await poponta(message)

File: my_commands/in_command.pl
#!/usr/bin/perl
use strict;
use warnings;
use utf8;
use open IN  => ":utf8";
use open OUT => ":utf8";
use open IO => ":utf8";

my @command_module_files = grep {$_ ne "my_commands/in_commands/__init__.py"} (glob "my_commands/in_commands/*.py");
my @functions;
my %func2str;
my %file2function;
foreach my $command_module_file (@command_module_files){
  open IN, '<', $command_module_file;
  my $functions_line = <IN>;
  $functions_line =~ s/\x0D?\x0A?$//;
  $functions_line =~ s/^#//;
  $file2function{$command_module_file} = $functions_line;
  my $str = <IN>;
  $str =~ s/\x0D?\x0A?$//;
  $str =~ s/^#//;
  $func2str{$functions_line} = $str;
  @functions = (@functions, $functions_line);
  close IN;
}
open FH, '>', "my_commands/in_command.py";
print FH "#in_command\nimport discord\n";
foreach my $command_module_file (@command_module_files){
  my $command_module = $command_module_file;
  $command_module =~ s/\//./g;
  $command_module =~ s/.py$//;
  print FH "from $command_module import $file2function{$command_module_file} \n";
}

print FH "async def in_command(message:discord.Message):\n";
foreach my $function (@functions){
  my $str2 = $func2str{$function};
  print FH "    if ${str2} in message.content:\n";
  print FH "        await ${function}(message)\n";

}
close FH;
